File: jeorg-camel-app-atm/src/main/java/com/jesperancinha/atm/finder/service/payload/response/ATMGeoLocation.java
package com.jesperancinha.atm.finder.service.payload.response;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Getter;
import lombok.Setter;

/**
 * Created by joaofilipesabinoesperancinha on 28-07-16.
 */
@Getter
@Setter
@JsonIgnoreProperties(ignoreUnknown = true)
public class ATMGeoLocation {

    @JsonProperty("lat")
    private Double lat;

    @JsonProperty("lng")
    private Double lng;
}

File: jeorg-camel-app-atm/src/main/kotlin/org/jesperancinha/atm/finder/service/payload/response/ATMGeoLocation.kt
package org.jesperancinha.atm.finder.service.payload.response

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonProperty

/**
 * Created by joaofilipesabinoesperancinha on 28-07-16.
 */
@JsonIgnoreProperties(ignoreUnknown = true)
data class ATMGeoLocation(
    @JsonProperty("lat")
    val lat: Double,

    @JsonProperty("lng")
    val lng: Double,
)
